docker/volumeplugin/rancher.go

```diff
@@ -108,6 +110,24 @@ func isCreated(driver string, vol client.Volume) bool {
 	return goodStates[vol.State]
 }
 
+func (r *RancherState) GetAny(name string) (*volume.Volume, *client.Volume, error) {
```

Comment:
I don't think this needs to be a public method because it's used only internally